schemas/includes/company-for-nesting.json

```diff
@@ -85,6 +85,10 @@
     "mailing_address": {
       "$ref": "address.json"
     },
+    "address": {
```

Comment:
Convert to array of "address_with_type" objects. Start/end dates can/will be added later.